pkg/apis/monitoring/v1alpha1/types.go

```diff
@@ -156,6 +156,8 @@ type PrometheusConfig struct {
 	// Number of replicas/pods to deploy for a Prometheus deployment.
 	// +optional
 	// +kubebuilder:default=2
+	// +kubebuilder:validation:Required
```

Comment:
doesn't this conflict with the `//optional` above. 
`replicas` is optional and defaults to `2`. I think only `validation:Minimum` is what we need

Reply:
Hmm maybe....let me try

Reply:
It seems to work exactly the same with `kubebuilder:validation:Required` as well as without it. Do you want me to remove it?

Reply:
If you look at the CRD manifest, the `Required` doesn't actually do anything. That's likely because we have a default. So :+1: for dropping this annotation.